### PR TITLE
USAGOV-56 integrate analytics backend

### PR DESCRIPTION
**Pull request by jacobaaronyeager**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-56

## Checklist for the Developer

- [ ] A link to the JIRA ticket has been added above.
- [ ] The JIRA ticket identifies the desired result of this change.
- [ ] The JIRA ticket contains clear acceptance criteria.
- [ ] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [ ] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: .docker/src-reporter/etc/ssl/openssl.cnf.bak
Comment (akf) [inline .docker/src-reporter/etc/ssl/openssl.cnf.bak:1]: Do we need openssl.cnf.bak? Where is this from?

File: bin/cloudgov/setup-egress-for-apps [hunk 1]
```diff
@@ -141,10 +141,7 @@ $output cf target -o "$startorg" -s "$startspace" 2>&1 > /dev/null
 for app in $applist; do
     echo "Configure Egress settings for $app"
     $output cf add-network-policy "$app" "$proxyname" -s "$egressspace" -o "$egressorg" --protocol tcp --port 61443
-    $output cf unset-env "$app" http_proxy  # $proxyroute
```
Comment (akf) [inline bin/cloudgov/setup-egress-for-apps:144]: Just as a heads-up, we added these lines to explicitly clear these http[s]_proxy environment variables when we realized they were messing up Drupal's ability to retrieve s3 files. 

It's fine to delete these lines as long as we don't have something else setting them again. If we've missed something like that somewhere, problems with newly-uploaded images will be the symptom.

File: bin/cloudgov/setup-egress-for-space [hunk 2]
```diff
@@ -92,27 +92,27 @@ if [ -n "${DRYRUN}" ]; then
...
+#         BUCKET=$(            echo -E "$VCAP_SERVICES" | jq -r ".s3[$i].credentials.bucket")
+#         AWS_ENDPOINT=$(      echo -E "$VCAP_SERVICES" | jq -r ".s3[$i].credentials.endpoint" | uniq )
+#         AWS_ENDPOINT_ALT=$(  echo -E "$AWS_ENDPOINT"  | sed 's/s3\-us\-/s3.us-/g' | uniq )
+#         AWS_FIPS_ENDPOINT=$( echo -E "$VCAP_SERVICES" | jq -r ".s3[$i].credentials.fips_endpoint" | uniq )
+#         # Add both the FIPS and non-FIPS hostnames
+#         buckethosts="$buckethosts ${BUCKET}.${AWS_ENDPOINT} ${BUCKET}.${AWS_ENDPOINT_ALT} ${BUCKET}.${AWS_FIPS_ENDPOINT}"
+#         ((i+=1))
+#     done
+# done
+# echo "Using buckets: $buckethosts"
```
Comment (akf) [inline bin/cloudgov/setup-egress-for-space:115]: $buckethosts is used later in this file, on line 131. Probably that's more code that can be commented out. 

I don't know why I'm like this.

**Review by akf — CHANGES_REQUESTED** [review]
I have a concern that not setting "bucketfiles" in setup-egress-for-space might break the script -- see my note there. (I didn't run it to check, obviously.) 

Otherwise, it looks ready to merge. I would very much like to get a quick run-through from you on the port changes, though -- basically everywhere you changed something to or from "61443". I'll bet this was a big chunk of the problem!

File: bin/cloudgov/setup-egress-for-space (hunk 2 above)
Comment (jacobaaronyeager) [reply bin/cloudgov/setup-egress-for-space:115]: I appreciate you.

**Review by akf — APPROVED** [review]

**Pull request merged**